README.md

```diff
@@ -74,12 +183,19 @@ services:
       test:
         commands:
           - ./util/install-docker-compose.sh
-          - docker-compose -f docker-compose-test.yml up --abort-on-container-exit
+      steps:
+        deploy:
+          config:
+            includes: ../src/modules/scala/sample-cluster/kubernetes/servicer-akka-cluster.yaml
```

Comment:
I think it would be good to expand the include here to show the basics for the kube_cluster adapter:

```
steps:
  deploy:
    config:
      cluster_name: my_gke_cluster
      project: my_gcloud_project
      compute_zone: us-central1-f
      wait_for_pending: true
      cluster_configs:
        - ...    # kubernetes config goes here
```

Reply:
good call